modules/page-header/react/lib/PageHeader.tsx

```diff
@@ -31,10 +31,10 @@ const Container = styled('div')<PageHeaderProps>(
     overflow: 'hidden',
     padding: `0 ${spacing.s}`,
   },
-  ({marketing, breakpoints}) => {
+  ({capWidth, breakpoints}) => {
```

Comment:
I forgot that the marketing version had different padding at larger sizes than the normal version. This doesn't make sense when we change it to `capWidth` and I don't believe it's necessary anymore.

Can we change the `breakpoints` prop to `breakpoint?: number` instead and just have one media query (regardless of `capWidth`)?

Reply:
@anicholls should be good for another look– we're using a single breakpoint now that defaults to `768`.